plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -35,10 +35,23 @@ import (
 )
 
 const (
+	bucketRange        = `(0,1,2,3,4,5,6,7,8,9)`
```

Comment:
This makes every single constant query into a var that gets created in the init file. Are we ok with this?